deploy/helm/kubernetes/templates/kubedns-manifests.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.coredns.enabled }}
+{{- $fullName := include "kubernetes.fullname" . -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $fullName }}-coredns-manifests
+data:
+  {{- if .Values.coredns.enabled }}
+  coredns.yaml: |
+    {{- tpl (.Files.Get "manifests/coredns-1.8.yaml") . | nindent 4 }}
```

Comment:
Is there any specific reason for specifying version in filename?

Reply:
> It depends on whether you'd like to support several CoreDNS versions simultaneously

How are two coredns versions can act together in single cluster?

> CoreDNS version does not have strict mapping on Kubernetes version in fact.

That's true, but it has some defaults in kubeadm:
https://github.com/kubernetes/kubernetes/blob/5e58841cce77d4bc13713ad2b91fa0d961e69192/cluster/addons/dns/coredns/coredns.yaml.base#L134-L135